frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
```

Comment:
Where? Diagram cache preview

Issues:
- Diagram preview title should be larger & bold
- There are scrollbars verticals and horizontal


![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/1998d44b-7ef7-486f-ad70-503476d7536d)